### DISCUSSION TITLE
403 when init

### DISCUSSION THREAD

**Discussion by kud** [ankitpokhrel/jira-cli#735] [Q&A]
Hey 👋

When runing `jira init` on a cloud, I always have that:

```
⠴ Verifying login details...

REQUEST DETAILS
------------------------------------------------------------

GET /rest/api/2/myself HTTP/1.1
Host: theorchard.atlassian.net
Authorization: Bearer {Hidden Token}


✗ Received unexpected response '403 Forbidden' from jira. Please try again.
```

I've tried two times to create this token, still nothing.

Any idea?

Thanks 😌

**Comment by maftieu** [comment]
Did you set `JIRA_AUTH_TYPE` to "bearer"?
Try to not set it. See https://github.com/ankitpokhrel/jira-cli/issues/746

**Reply by kud** [reply]
Indeed, setting `JIRA_AUTH_TYPE` to empty (and really assigning to empty) did the trick. Thanks!

**Reply by goatatwork** [reply]
@maftieu Thank you for this!! Totally worked. I was about to give up on jira-cli until I found this. ❤️
